tailwind.config.js

```diff
@@ -0,0 +1,105 @@
+// tailwind.config.js
+
+import tailwindcssAnimate from "tailwindcss-animate";
+import { colorOpacityPlugin } from "./src/lib/plugins/colorOpacityPlugin";
+import colors from "tailwindcss/colors";
+
+/** @type {import("tailwindcss").Config} */
```

Comment:
we can use tailwind.config.ts to get full type support instead of this type comment

Reply:
The JS type documentation also provides full type support. Therefore, both TS and JS can work.